lib/radiator_web/graphql/admin/resolvers/editor.ex

```diff
@@ -105,7 +105,13 @@ defmodule RadiatorWeb.GraphQL.Admin.Resolvers.Editor do
   end
 
   def find_user(_, _, %{context: %{current_user: user}}) do
-    {:ok, user}
+    user
+    |> Radiator.Repo.preload(:person)
+    |> (&{:ok, &1}).()
```

Comment:
Might be worth considering adding a helper function that wraps anything in an :ok tuple for readability. If we can come up with a good name.

Reply:
Yeah, Like Tuple.ok(). Kind of dislike the ugliness of this statement here, but like that it is concise.